src/components/Files/PreUploadModal.tsx

```diff
@@ -100,7 +99,10 @@ export const PreUploadDialog = ({
     if (allowedTypes.includes('*/*')) {
       return [t('all')];
     }
-    return allowedTypes.map((mimeType) => extension(mimeType));
+    return allowedTypes
+      .map((mimeType) => getExtensionsList(mimeType))
+      .flat()
+      .map((type) => `.${type}`);
```

Comment:
it's good candidate for util-method because the same block in `FileManagerModal.tsx`